src/Client/index.ts

```diff
@@ -18,14 +18,31 @@ import { debug } from "../utils";
 import { discoverEndpoint } from "./discovery";
 import { parseConnectionString } from "./parseConnectionString";
 
-export interface DNSClusterOptions {
-  discover: EndPoint;
+interface DiscoveryOptions {
```

Comment:
**SIDENOTE:** It's not a blocker but in .NET client, those settings have Settings suffix: https://github.com/EventStore/EventStore-Client-Dotnet/blob/master/src/EventStore.Client/EventStoreClientConnectivitySettings.cs